### PR TITLE
[WIP] Endpoint for tags

### LINKED ISSUE #34: [FEATURE-REQUEST] Add endpoint for Tags
**Please Describe The Problem To Be Solved**
Please see specificatiosn sheet for details on `/tag` endpoints.

**Suggest A Solution or Implementation**

**Screenshots**

### PR DESCRIPTION
**Pull request by thescriptninja**
# Changes proposed through this PR
  
- Controller for tag endpoint added
- DTO for tags added

**Before submitting this PR, make sure:**

- [x] There is an issue corresponding the changes in this PR.
- [x] The base branch for this PR is master and not based on another PR.
- [x] The changes made are independent from the changes made in other PR's.
- [x] You have documented the changes made and updated the README file.
- [x] Merging this PR doesn't disrupt functionality or cause a feature to behave unexpectedly.
- [x] PR is ready to be merged, otherwise you've added WIP in the title.

**Attach screenshots that might help in the review:**

- NONE

---
fixes #34

### CODE REVIEW

**Comment by thescriptninja** [comment]
Will add more routes in future commits. @thealphadollar please review

File: app/main/controller/tag_controller.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Routes
+
+@api.route('/tag/getAll')
+class AllTags(Resource):
+    @api.doc('list of all tags')
+    @api.marshal_list_with(_tag)
+    def get(self):
+        """Gets all tags"""
+        # Make db call
```
Comment (thealphadollar) [inline app/main/controller/tag_controller.py]: Why have you left this? There are methods already present for the same.

Comment (thescriptninja) [reply app/main/controller/tag_controller.py]: Yeah I missed #31. Will do this once it is merged

**Comment by thealphadollar** [comment]
@thescriptninja Kindly make the requested changes.

**Comment by thealphadollar** [comment]
@thescriptninja Please resolve conflicts :)

**Comment by thealphadollar** [comment]
@thescriptninja Please do not send PRs from your master branch, I cannot push to them and hence I cannot make changes in yiour PR and this makes us slow :cry:

**Pull request merged**